src/Features/LanguageServer/Protocol/LspServices/LspServices.cs

```diff
@@ -44,17 +45,22 @@ internal class LspServices : IDisposable
         _lazyLspServices = services.ToImmutableDictionary(lazyService => lazyService.Metadata.Type, lazyService => lazyService);
     }
 
-    public T GetRequiredService<T>() where T : class, ILspService
+    public T GetRequiredLspService<T>() where T : class, ILspService
+    {
+        return GetRequiredService<T>();
+    }
+
+    public T GetRequiredService<T>()
     {
         var service = GetService<T>();
         Contract.ThrowIfNull(service, $"Missing required LSP service {typeof(T).FullName}");
         return service;
     }
 
-    public T? GetService<T>() where T : class, ILspService
```

Comment:
Other ILspServices implementations will not have the `ILspService` requirement so we have to wrap this a bit.